src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2771,6 +2991,12 @@ void processContextElement
     //
     releaseTriggeredSubscriptions(subsToNotify);
 
+    // FIXME P10: I would say that the memory allocated previously for notifyCerP should
+    // be released, but uncomentaing this line produces a double-free crash. Mabye somebody
+    // has already released before?
+    //notifyCerP->release();
+    //delete notifyCerP;
```

Comment:
I understand that `delete notifyCerP` should be used here. However, I'm having some problems doing so... maybe something that I don't understand (or is wrong) in the chained releasing of the internal objects.